src/lib/codap-helper.ts

```diff
@@ -8,6 +8,8 @@ export interface DataContext {
 
 const dataContextResource = (contextName: string, subKey?: string) =>
                               `dataContext[${contextName}]${subKey ? "." + subKey : ""}`;
+const collaboratorsResource = (contextName: string, subKey: string) =>
+                                `dataContext[${contextName}].collection[Collaborators]${subKey}`;
```

Comment:
Apparently this works, but isn't it supposed to have a period after `collection[Collaborators]`? 

This creates resource identifiers in the form `dataContext[collaborative-table].collection[Collaborators]case` and `dataContext[collaborative-table-1].collection[Collaborators]caseByIndex[0]`. But CODAP seems not to care. (Adding the dot also works.)